chunky/src/java/se/llbit/chunky/renderer/ModifiableRenderOptions.java

```diff
@@ -0,0 +1,40 @@
+package se.llbit.chunky.renderer;
+
+public class ModifiableRenderOptions implements RenderOptions {
```

Comment:
See above. Can we improve this so that modifying the render options is safe and gets applied after the user confirms (or reset otherwise)?

Reply:
Pretty sure that this would be the job of the UI logic or something in the render code which copies its state from this object. At the moment, the user can only set the `tile size` and `spp per pass` via CMD argument, right? And render thread count had some special handling somewhere which already required restarting the rendering or Chunky (?). Definitively a useful suggestion for a follow-up issue + PR :)